### PR TITLE
Update directory in Build Config RFC

### PR DESCRIPTION
**Pull request by AidanDelaney**
We are triangulating this RFC, the implementation and the platform specification.  This change makes the RFC agree with the implementation.

Signed-off-by: Aidan Delaney <adelaney21@bloomberg.net>

### CODE REVIEW

File: text/0109-build-config.md [hunk 1]
```diff
@@ -40,10 +40,10 @@ The environment variables may ideally also take precendence over any user provid
 # What it is
 [what-it-is]: #what-it-is
 
-The RFC proposes the introduction of the following directory `/cnb/config/env.build` in build images. The directory follows the same convention as a `CNB environment directory`. The notable difference is that the environment variables sourced from this directory are applied **AFTER** processing the user-provided platform environment variables i.e. they should have the highest precedence. These variables should be available during both `detect` and `build` phases (and the `generate` phase in the future).
+The RFC proposes the introduction of the following directory `/cnb/build-config/env.build` in build images. The directory follows the same convention as a `CNB environment directory`. The notable difference is that the environment variables sourced from this directory are applied **AFTER** processing the user-provided platform environment variables i.e. they should have the highest precedence. These variables should be available during both `detect` and `build` phases (and the `generate` phase in the future).
```
Comment (natalieparellano) [inline text/0109-build-config.md:43]:
```suggestion
The RFC proposes the introduction of the following directory `/cnb/build-config/env` in build images. The directory follows the same convention as a `CNB environment directory`. The notable difference is that the environment variables sourced from this directory are applied **AFTER** processing the user-provided platform environment variables i.e. they should have the highest precedence. These variables should be available during both `detect` and `build` phases (and the `generate` phase in the future).
```

Comment (natalieparellano) [reply text/0109-build-config.md:43]: (per https://github.com/buildpacks/spec/pull/345#discussion_r1063589414, unless we want to change the implementation)

File: text/0109-build-config.md [hunk 2]
```diff
@@ -40,10 +40,10 @@ The environment variables may ideally also take precendence over any user provid
 # What it is
 [what-it-is]: #what-it-is
 
-The RFC proposes the introduction of the following directory `/cnb/config/env.build` in build images. The directory follows the same convention as a `CNB environment directory`. The notable difference is that the environment variables sourced from this directory are applied **AFTER** processing the user-provided platform environment variables i.e. they should have the highest precedence. These variables should be available during both `detect` and `build` phases (and the `generate` phase in the future).
+The RFC proposes the introduction of the following directory `/cnb/build-config/env` in build images. The directory follows the same convention as a `CNB environment directory`. The notable difference is that the environment variables sourced from this directory are applied **AFTER** processing the user-provided platform environment variables i.e. they should have the highest precedence. These variables should be available during both `detect` and `build` phases (and the `generate` phase in the future).
```
Comment (AidanDelaney) [inline text/0109-build-config.md:43]:
```suggestion
The RFC proposes the introduction of the following directory `/cnb/build-config/env.build` in build images. The directory follows the same convention as a `CNB environment directory`. The notable difference is that the environment variables sourced from this directory are applied **AFTER** processing the user-provided platform environment variables i.e. they should have the highest precedence. These variables should be available during both `detect` and `build` phases (and the `generate` phase in the future).
```

File: text/0109-build-config.md [hunk 3]
```diff
@@ -12,7 +12,7 @@
 # Summary
 [summary]: #summary
 
-This RFC proposes an easy way to configure build images to allow specifying a `/cnb/config/env.build` CNB environment directory that allows updating the Buildpack `detect` and `build` environment based on the directory.
+This RFC proposes an easy way to configure build images to allow specifying a `/cnb/build-config` CNB environment directory that allows updating the Buildpack `detect` and `build` environment based on the directory.
```
Comment (AidanDelaney) [inline text/0109-build-config.md:15]:
```suggestion
This RFC proposes an easy way to configure build images to allow specifying a `/cnb/build-config/env.build` CNB environment directory that allows updating the Buildpack `detect` and `build` environment based on the directory.
```

**Review by samj1912 — APPROVED** [review]

**Pull request merged**